modules/core/nuxeo-core-api/src/test/java/org/nuxeo/ecm/core/api/pathsegment/PathSegmentServiceTest.java

```diff
@@ -91,7 +92,7 @@ public void testContrib() throws Exception {
     @Test
     @Deploy("org.nuxeo.ecm.core.api.tests:OSGI-INF/test-pathsegment-contrib.xml")
     public void testContribOverride() throws Exception {
-        PathSegmentService service = Framework.getService(PathSegmentService.class);
+        assertNotNull(service);
```

Comment:
Not needed, test framework will throw if it cannot inject. Same in other methods.